src/cascadia/TerminalApp/defaults.json

```
@@ -0,0 +1,214 @@
+// THIS IS AN AUTO-GENERATED FILE! Changes to this file will be ignored.
+{
+    "alwaysShowTabs": true,
+    "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
+    "initialCols": 120,
+    "initialRows": 30,
+    "requestedTheme": "system",
+    "showTabsInTitlebar": true,
+    "showTerminalTitleInTitlebar": true,
+    "wordDelimiters": " ./\\()\"'-:,.;<>~!@#$%^&*|+=[]{}~?\u2502",

```

nit:

```suggestion
    "wordDelimiters": " /\\()\"'-.,:;<>~!@#$%^&*|+=[]{}~?\u2502",
```

Minor reordering and removing the double `.`

reject if no other changes